jwtauthenticator/jwtauthenticator.py

```diff
@@ -12,30 +12,51 @@ def get(self):
         header_name = self.authenticator.header_name
         auth_header_content = self.request.headers.get(header_name, "")
         signing_certificate = self.authenticator.signing_certificate
+        secret = self.authenticator.secret
         username_claim_field = self.authenticator.username_claim_field
         audience = self.authenticator.expected_audience
+        tokenParam = self.get_argument('token', default=False)
```

Comment:
Apologies for delay - been in dublin for the weekend and spent today suffering in work for it!

Agreed on the above, when you've made the changes i'll review and merge :) 

in terms of OAuth, you make a good point about there being a conflict. I wonder whether "Authorization" is a good default, since that's what's in the RFC for passing tokens in the header. I'll let you decide :)